nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/linalg/api/ops/impl/scalar/comparison/ScalarLessThan.java

```diff
@@ -65,7 +65,7 @@ public String onnxName() {
 
     @Override
     public String tensorflowName() {
-        return "less";
+        return "Less";
```

Comment:
This mapping is incorrect, and already mapped to LessThan custom op - remove method.

Reply:
Fixed.